### PR TITLE
Penn Rides data added. Changes format of transit/routes.

### PR DESCRIPTION
**Pull request by awiss**

### CODE REVIEW

File: server/transit.py [hunk 1]
```diff
@@ -149,7 +201,15 @@ def populate_route_info():
           routes[route_name] = [to_insert]
   for route in routes:
     routes[route] = sorted(routes[route], key= lambda stop: stop["order"])
+
+
+  routes['Campus Loop'] = routes['Campus Loop'][:-1]
```
Comment (adelq) [inline server/transit.py]: Can you add a comment why this is necessary?

Comment (adelq) [inline server/transit.py]: I know it's in the docstring, but still...

File: server/transit.py [hunk 2]
```diff
@@ -149,7 +201,15 @@ def populate_route_info():
...
+  # Filter out bad routes
   good_routes = ['PennBUS East', 'PennBUS West', 'Campus Loop']
-  routes = {key: routes[key] for key in routes if key in good_routes}
+
+  routes = [{'route_name': key, 'stops': routes[key]} for key in routes if key in good_routes]
+
+  # This last stop is erroneous, so manually taking it out
```
Comment (adelq) [inline server/transit.py]: I guess this comment was supposed to go above...

File: server/transit.py [hunk 3]
```diff
@@ -94,29 +116,50 @@ def add_path_points(l, stop):
...
+      stop['path_to'] = []
+
+      # margin of error is necessary to identify which waypoints are actually stops
+      while abs(all_waypoints[i]["Latitude"] - stop["Latitude"]) > 0.0001:
+
+        stop['path_to'].append(all_waypoints[i])
+        print all_waypoints[i], 'added to path'
+        i += 1
+        if i >= len(all_waypoints):
+          raise ValueError('pennrides and ISC Data do not match')
```
Comment (adelq) [inline server/transit.py]: lol

**Comment by awiss** [comment]
I am merging this. Nits are fixed and this should go up before Sunday.

**Pull request merged**